db/migrate/20180424194716_add_description_to_posts.rb

```diff
@@ -0,0 +1,7 @@
+class AddDescriptionToPosts < ActiveRecord::Migration[5.1]
+  def change
+    add_column :posts, :description, :text
+    add_column :posts, :posted_at, :datetime
```

Comment:
This is useless, use `created_at`

Reply:
That should already be removed in a different db action.